### DISCUSSION TITLE
Feature Request: 希望能够拖动thino卡片进入白板后，直接建立块引用的卡片

### DISCUSSION THREAD

**Discussion by kine56** [Quorafind/Obsidian-Thino#726] [Ideas 想法]
### Feature Requested | 请求的功能

目前thino的卡片是可以拖拽的，拖到其他地方就是复制卡片内容，拖动入白板内也是如此。
希望可以实现拖动卡片到白板内，直接将该卡片以“块引用”的形式呈现，这样做的好处是，可以将积累的卡片在白板内进行整理分类等，而无需再复制到白板内一份，避免数据重复。
虽然也可以用obsidian自带的块引用“[[^^关键词]]”来实现，但是这样一个一个关键词来搜索匹配效率太低了。
谢谢开发者！

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

[a comment by an automated account is not shown]

**Comment by runningZ1** [comment]
我有同样有这样的需求，希望开发者跟进

[a comment by an automated account is not shown]
